app/models/project.rb

```diff
@@ -0,0 +1,5 @@
+class Project < ApplicationRecord
+  validates :title, :date, :description, presence: { message: "must be given please" }
+  validates :title, length: { maximum: 100, too_long: "%{count} characters is the maximum allowed" }
```

Comment:
Why have you set the limit to 100 chars? No need imho, i would drop this validation personally.